frontend/src/components/assessment-creation/CreateAssessmentPage.tsx

```diff
@@ -0,0 +1,241 @@
+                    ...provided,
+                    color: "grey.300",
+                  }),
+                }}
+              />
+              <FormErrorMessage>{error?.message}</FormErrorMessage>
+            </FormControl>
+          )}
+        />
+      </Box>
+
+      <Box width="50%">
+        <Controller
+          control={control}
+          name="assessmentType"
+          rules={{ required: "Please select a type of assessment" }}
+          render={({
+            field: { onChange, value, name, ref },
+            fieldState: { error },
+          }) => (
+            <FormControl isRequired isInvalid={Boolean(error)}>
+              <FormLabel color="grey.400"> Type of Assessment </FormLabel>
+              <RadioGroup
+                name={name}
+                ref={ref}
+                onChange={onChange}
+                value={value}
+              >
+                <Stack direction="column">
+                  <Radio value="beginning" _checked={{ iconColor: "green" }}>
+                    Beginning of Grade
+                  </Radio>
+                  <Radio value="end">End of Grade</Radio>
+                </Stack>
+              </RadioGroup>
+              <FormErrorMessage>{error?.message}</FormErrorMessage>
+            </FormControl>
+          )}
+        />
+      </Box>
+
+      <FormControl>
+        <Text textStyle="subtitle2" mb="2">
+          {" "}
+          Curriculum{" "}
+        </Text>
+        <Stack direction="row" width="100%">
+          <Controller
+            control={control}
+            name="country"
+            rules={{ required: "Please select a country" }}
+            render={({
+              field: { onChange, onBlur, value, name, ref },
+              fieldState: { error },
+            }) => (
+              <FormControl isRequired isInvalid={Boolean(error)} mr={2}>
+                <FormLabel color="grey.400"> Country </FormLabel>
+                <Select
+                  name={name}
+                  ref={ref}
+                  onChange={onChange}
+                  onBlur={onBlur}
+                  value={value}
+                  options={countryOptions}
+                  placeholder=""
+                  chakraStyles={{
+                    dropdownIndicator: (provided) => ({
+                      ...provided,
+                      bg: "transparent",
+                      px: 2,
+                      cursor: "inherit",
+                    }),
+                    indicatorSeparator: (provided) => ({
+                      ...provided,
+                      display: "none",
+                    }),
+                    placeholder: (provided) => ({
+                      ...provided,
+                      color: "grey.300",
+                    }),
+                  }}
+                />
+                <FormErrorMessage>{error?.message}</FormErrorMessage>
+              </FormControl>
+            )}
+          />
```

Comment:
Is this the exact same country selection as in the `TeacherSignup`? If so, can we refactor to a common component instead of duplicating the code?